### DISCUSSION TITLE
integration of eUPF with OAI core

### DISCUSSION THREAD

**Discussion by kcl17** [edgecomllc/eupf#464] [Ideas]
Are you planning to integerate eupf with other 3gpp 5g cores like OAI .

**Comment by ewdss** [comment]
Hi! Yes, we are going to do it in the next few sprints. Can you explain what is your expectations about specific OAI or some other cores? And can you propose any other core for the integration except OIA?

**Comment by Satyam-git-hub** [comment]
hello @ewdss 
you can try ONF sd-core
